### PR TITLE
PS-6783 [Doc] Release Notes

### PR DESCRIPTION
**Pull request by patrickbirch**
merged with 5.6 version
removed files
Percona-Server-5.6.47-87.0
Percona-Server-5.6.40-84.0
Percona-Server-5.6.44-85.0
Percona-Server-5.6.45-86.1
modified files
release-notes_index.rst
conf.py
added file
Percona-Server-5.7.29-32

### CODE REVIEW

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-5.7.29-32.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.29-32
+includes all the features and bug fixes available in
+`MySQL 5.7.29 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-29.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-1469`: The Memory storage engine detected an incorrect "is full" condition when the space contained reusable memory chunks and could be reused.
```
Comment (kamil-holubicki) [inline doc/source/release-notes/Percona-Server-5.7.29-32.rst]: reusable memory chunks THAT could be reused

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.29-32.rst]: Fixed with "that"

**Review by hrvojem — COMMENTED** [review]
I know this is still WiP concerning the automation, but code formatting of commands looks much nicer and makes more sense when being read. I mean things like:
* SHOW ENGINE INNODB STATUS
* INSERT ... ON DUPLICATE KEY UPDATE
* group_replication.gr_majority_loss_restart
* innodb_zip.wl5522_zip
* innodb.alter_missing_tablespace
and few others

**Review by kamil-holubicki — APPROVED** [review]

**Review by hrvojem — APPROVED** [review]
LGTM

**Pull request merged**